lib/main.js

```diff
@@ -33,11 +41,19 @@ module.exports = {
   activate() {
     require('atom-package-deps').install('linter-jshint');
 
-    this.scopes = ['source.js', 'source.js-semantic'];
+    this.scopes = [];
```

Comment:
Actually, now that the array reference is being properly kept, there _is_ need of this line due to the ordering of how it is set.